app/src/main/AndroidManifest.xml

```diff
@@ -22,16 +23,24 @@
 		android:icon="@drawable/ic_launcher"
 		android:label="@string/app_name"
 		android:supportsRtl="true"
+		android:usesCleartextTraffic="true"
 		android:theme="@style/Theme.TVLauncher">
-		<activity android:name=".LauncherActivity">
-			<intent-filter>
+		<activity
+			android:name=".LauncherActivity"
+			android:stateNotNeeded="true"
+			android:excludeFromRecents="true"
+			android:launchMode="singleTask"
+			android:screenOrientation="landscape">
```

Comment:
Both landscape and portrait orientations work so we don't need to force landscape mode, so this option is not needed.